ElectronClient/app/gui/ConfigScreen.jsx

```diff
@@ -61,7 +61,7 @@ class ConfigScreenComponent extends React.Component {
 			const settings = Object.assign({}, this.state.settings);
 			settings[key] = value;
 			this.setState({ settings: settings });
-		}
+		};
```

Comment:
Please don't add or remove this kind of optional character as it makes the commit needlessly big.